### ISSUE TITLE
check dating of 2 V H.23 

### ISSUE THREAD

**Issue by dietmarpravida** [faustedition/faust-xml#671]
nicht vor 1825-02-27 | nicht nach 1825-02-26

http://dev.faustedition.net/macrogenesis/2_V_H.23

**Comment by dietmarpravida** [comment]
@thvitt Das ist ein besonders interessanter Fall. Zur Demonstration der Arbeit des Algorithmus, ließe sich sich das sehr gut heranziehen. Dennoch ist das Ergebnis "nicht vor 1825-02-27 | nicht nach 1825-02-26" offensichtlich fehlerhaft. Wird das Datum 27. irgendwie errechnet? Es stammt ja nicht direkt aus den Daten, oder? Dass das Ergebnis einen Widerspruch darstellt (wenn man nicht 26.02.1826, 24:00 Uhr annehmen möchte), wird aber anscheinend nicht bemerkt.

**Comment by thvitt** [comment]
ja, ist ein interessanter fall, mal den pfad zu den daten angeben lassen
ich tippe ja auf ±1-fehler irgendwo

Die datierung wird errechnet, indem aus dem DAG (ohne Konflikt- und ignorierte Kanten) von allen Daten, von denen aus der Zeuge erreichbar ist, das späteste sowie von allen Daten, die vom Zeugen erreichbar sind, das früheste genommen wird (so dass eine möglichst enge Datierung erreicht wird). dann wird noch ein tag weiter gerechnet um aus vor ein nicht früher als zu machen

müsste mal die pfade zu diesen daten visualisieren

**Comment by dietmarpravida** [comment]
@thvitt Fall für dhd?

**Comment by thvitt** [comment]
Die Datierung landet [ja jetzt](http://dev.faustedition.net/macrogenesis/2_V_H.23) bei nicht vor & nicht nach (also am) 1825-02-26, was da in 1.1 schief lief, seh ich grad nicht
